modules/auxiliary/scanner/http/wordpress_cp_calendar_sqli.rb

```diff
@@ -0,0 +1,74 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'uri'
+require 'msf/core'
+
+class Metasploit4 < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpClient
```

Comment:
@brandonprry jeah there is no benefit in this module when using the wordpress mixin. But you could include it and write a check method calling `check_plugin_version_from_readme` (https://github.com/rapid7/metasploit-framework/blob/master/lib/msf/http/wordpress/version.rb#L50). But as this module only checks for the SQLI without exploiting it, the main method already acts as a "check method".